docker/local/Dockerfile.local

```diff
@@ -1,4 +1,5 @@
 ARG DEVICE=gpu
```

Comment:
Have you got any number about time to build the docker image here? I think a big burden here is that the time for building parmetis docker image will be too long, and also make the docker image itself too large.

Reply:
ParMETIS dependencies are or added if requested by the user, the build time should be roughly the same as building the ParMETIS image using the specialized Dockerfile. 

Could you clarify if there's a concern I'm missing here? I will add some measurements of build times for the images using the specialized Dockerfile  vs. this one.